### ISSUE TITLE
Adding more images to dataset

### ISSUE THREAD

**Issue by mronian** [JuliaImages/TestImages.jl#17]
I'd like to add some 16bit images to the dataset. One of the sources I found is this link => http://www.brucelindbloom.com/index.html?ReferenceImages.html.

The author requires that we use it for research/non-commercial purposes only. Can I add it to this dataset ?

**Comment by timholy** [comment]
I'm a bit reluctant to add encumbered images. Can you find some good alternatives? If not, I'm willing to discuss.

**Comment by mronian** [comment]
I found [these images](http://imagecompression.info/test_images/) (they are free to use), but they provide a huge zip file (grayscale is 285MB and RGB is 787MB). I would like to add just a few of these so that people can easily test without eating up a lot of their download. Is there a place we can host these images? This will also help to show images for ```jetplane, lake, livingroom``` etc from the standard dataset in the Readme.

There are some more images here are under a creative commons license => http://www.tecnick.com/public/code/cp_dpage.php?aiocp_dp=testimages

**Comment by timholy** [comment]
Wow, those are some big images. Clearly focused on digital photography.

I did notice they have a "thumbnail link", e.g., http://imagecompression.info/test_images/big_thumbnails/artificial.jpg. 

I suppose we could host an image or two here, I was just trying to cut down on the size of the repo.

**Comment by mronian** [comment]
How about we store the images in a proper website with documentation instead of in the ReadMe? I mean we could use the ```gh-pages``` branch to host the images (something like the documentation for Images.jl). This way the repo size will be small and we can also host the images.

**Comment by timholy** [comment]
I've never used (or even looked into) `gh-pages`, but that sounds like a great idea. The sources for some of these images tend to evaporate over time, as you surely noticed already.

**Comment by mronian** [comment]
Oh, I thought you might have as the docs of Images.jl are hosted in the ```gh-pages``` branch of the repo. Maybe Mk-docs automatically does that.

I've created a small website for TestImages.jl in the ```gh-pages``` branch of my fork. It will have to be merged in a new ```gh-pages``` branch here. Also once the website is merged, we can change the links in ```deps/build.jl``` to the new ones. I have opened a PR for that.
